packages/ui/src/prototypes/typography.stories.jsx

```
@@ -51,42 +49,24 @@ const ItemColorValue = ({ value }) => (
-
-    <Item colorName="dark" />
+    <div style={{ display: 'flex', marginBottom: '24px' }}>
+      <Item colorName="text" valueName="muted" />
+      <Item colorName="text" />
+      <Item colorName="text" valueName="intense" />
+    </div>
 
     <hr />
 
     {COLORS.map((colorName) => (
       <div style={{ display: 'flex', marginBottom: '24px' }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)